bundles/EcommerceFrameworkBundle/FilterService/FilterType/MultiSelectFromMultiSelect.php

```diff
@@ -17,13 +17,20 @@
 use Pimcore\Bundle\EcommerceFrameworkBundle\IndexService\ProductList\ProductListInterface;
 use Pimcore\Bundle\EcommerceFrameworkBundle\IndexService\Worker\WorkerInterface;
 use Pimcore\Bundle\EcommerceFrameworkBundle\Model\AbstractFilterDefinitionType;
+use Pimcore\Db;
 
 class MultiSelectFromMultiSelect extends SelectFromMultiSelect
 {
     public function getFilterValues(AbstractFilterDefinitionType $filterDefinition, ProductListInterface $productList, array $currentFilter): array
     {
         $field = $this->getField($filterDefinition);
-        $rawValues = $productList->getGroupByValues($field, true, !$filterDefinition->getUseAndCondition());
+
+        $useAndCondition = false;
```

Comment:
we should check `$filterDefinition instanceof \Pimcore\Model\DataObject\Fieldcollection\Data\FilterMultiSelectFromMultiSelect`, other wise throw `InvalidConfigException`